waspc/data/Generator/templates/server/src/routes/auth/me.ts

```diff
@@ -1,11 +1,10 @@
 import { serialize as superjsonSerialize } from 'superjson'
 import { handleRejection } from 'wasp/server/utils'
-import { createInvalidCredentialsError } from 'wasp/auth/utils'
 
 export default handleRejection(async (req, res) => {
   if (req.user) {
     return res.json(superjsonSerialize(req.user))
   } else {
-    throw createInvalidCredentialsError()
+    return res.json(null)
```

Comment:
```suggestion
    return res.json(superjsonSerialize(null))
```

Maybe this will get rid of the error `TypeError: payload is null`

Reply:
Yes!